lib/src/core/metadata/directives.dart

```diff
@@ -722,7 +727,7 @@ class DirectiveMetadata extends InjectableMetadata {
 ///
 /// ## Lifecycle hooks
 ///
-/// When the component class implements some [../../guide/lifecycle-hooks.html]
+/// When the component class implements some [lifecycle-hooks](docs/guide/lifecycle-hooks.html)
```

Comment:
The docs also appear on dartdocs.org, albeit somewhat broken.

What can we do to avoid bad links on dartdocs.org? Can we disable the Angular doc generation there, preferably with a page that says where the docs really live?

cc @astashov